### DISCUSSION TITLE
2023.32.1 won't boot

### DISCUSSION THREAD

**Discussion by paulharman** [tesla-android/issue-tracker#289] [General]
Having issues with 2023.32.1 - web interface just shows "Connection with Tesla Android services lost. The app will restart when it comes back"

Tried OTA update, SD flash and a different SD card.  Bootloader is updated.

**Comment by mikegapinski** [comment]
what address are you using in the browser? Are you connected to the TA Wi-Fi network directly?

**Reply by detion88** [reply]
The new version doesnt work with TA and tesla connection to router instead of directly to TA network?

**Reply by paulharman** [reply]
Didn't for me

**Reply by mikegapinski** [reply]
It would work if you redirect 104.248.101.213 to the address assinged to the pi on your router. I use iptables for this on the pi

**Reply by detion88** [reply]
Tried but dun seems to work. Will try again

**Comment by paulharman** [comment]
Ahhh I had connected via ethernet and gone to the device IP.  Just tried via the hotspot and it's working ok.  Thought it would work via ethernet as the initial page connected.
